spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkUtil.java

```diff
@@ -77,6 +77,8 @@ public class SparkUtil {
 
   private SparkUtil() {}
 
+  /** @deprecated will be removed in 1.4.0 */
```

Comment:
Great.  Maybe we can put a comment on why, and what the user can call.

Actually, I had a question, that I did not have a chance to ask in original pr: https://github.com/apache/iceberg/pull/7263, do we still need to close fileIO, at the end of action?  Is that getting handled somewhere?

Reply:
That's a good question and it impacts Spark jobs in general, not just this action, thus this is consistent with Iceberg behavior in general AFAIK. There has been some discussion on handling resource cleanup better, @danielcweeks @rdblue do you have any insights into that?

Reply:
I updated this and added comments about the deprecation.

Reply:
Makes sense.  I see the SparkActions takes the table, maybe it could be the responsibility of user then to close it, though it is a bit undocumented.